src/vivarium/interface/utilities.py

```diff
@@ -115,3 +118,23 @@ def get_output_root(results_directory, model_specification_file):
     model_name = Path(model_specification_file).stem
     output_root = Path(results_directory + f"/{model_name}/{launch_time}")
     return output_root
+
+
+def add_logging_sink(sink, verbose, colorize=False, serialize=False):
+    message_format = ('<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | '
+                      '<cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> '
+                      '- <level>{message}</level>')
+    if verbose:
+        logger.add(sink, colorize=colorize, level="DEBUG", format=message_format, serialize=serialize)
+    else:
+        logger.add(sink, colorize=colorize, level="ERROR", format=message_format, serialize=serialize)
+
+
+def configure_logging_to_terminal(verbose):
+    logger.remove()  # Clear default configuration
```

Comment:
Use logger.remove(0).  I think we just want to remove the default logger.  Otherwise I think this mucks up log handlers in the distributed tools.